generated/ridp/user_exists_in_enroll_IVL.json

```diff
@@ -0,0 +1,3 @@
+{
+  "error": "The social security number you entered is affiliated with another account."
```

Comment:
or even
```
{
  "error_type": "account_exists",
  "error_message": "The social security number you entered is affiliated with another account."
}
```
A human readable message is useful for a human trying to understand the response and could be displayed to the user, while a machine readable stable flag is nice for the phone to switch on